powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/WebApplicationConfig.java

```diff
@@ -37,9 +37,19 @@
 @Configuration
 public class WebApplicationConfig implements WebMvcConfigurer {
 
+    private final RESTResponseExceptionResolver restResponseExceptionResolver;
+
+    /**
+     * Configuration constructor.
+     * @param restResponseExceptionResolver REST response exception resolver.
+     */
+    public WebApplicationConfig(RESTResponseExceptionResolver restResponseExceptionResolver) {
```

Comment:
Aren't we missing the `@Autowired` annotation here, to make the code more readable?

Reply:
Added.